index.py

```diff
@@ -2,46 +2,59 @@
 
 import time
 from typing import Any, Dict, Optional
+import datetime
+from sqlalchemy import Column, Integer, DateTime
+
 
 from fastapi import FastAPI
 
 from typing import Optional
 
+import sqlalchemy as sa
+
 from sqlmodel import Field, Session, SQLModel, create_engine, select
+from pydantic import BaseModel
 
 class User(SQLModel, table=True):
     # TODO change the default value to the timestamp that the user is created
     id: Optional[int] = Field(default=None, primary_key=True)
     name: str
     secret_name: str
     age: Optional[str] = None
+    created_date: datetime.datetime = Field(sa_column=sa.Column(sa.DateTime(timezone=True), nullable=False))
       
 engine = create_engine("sqlite:///database.db")
 SQLModel.metadata.create_all(engine)
 # Create an instance of the API class
+user_1 = User(id=1, name="Peter", secret_name="Peter PArker", age=18)
 app = FastAPI()
 
+
+
 @app.get("/")
 async def root():
     # TODO include the user's name if they are logged in
-    return {"message": "Hello {}".format("World")}
+    return {"message": "Hello {}".format(user_1.name)}
+
+
 
 
 @app.post("/user")
-async def create_new_user(*, user: User):
+async def create_new_user(id: int, user: User):
```

Comment:
ID is currently auto generated by the database, what will happen when the user provides an ID of an existing user?